tests/mocket/src/main.rs

```diff
@@ -21,6 +21,7 @@ struct Opt {
     file_path: PathBuf,
 }
 
+#[allow(clippy::unused_io_amount)]
```

Comment:
Maybe this would be better on the smallest possible scope?
Perhaps right above ` SocketEvent::WaitForDisconnect => {` in the `match`?